utils/utils.py

```diff
@@ -0,0 +1,16 @@
+import logging
+import sys
+
+class Utils:
+    def __init__(self):
+
+        self.logger = logging.getLogger()
+        self.streamHandler = logging.StreamHandler(sys.stdout)
```

Comment:
```suggestion
        self.stream_handler = logging.StreamHandler(sys.stdout)
```

O correto ao se criar uma variavel em python é utilizar  o snake_case. Originalmente foi escrito com camelCase